### PR TITLE
tools/rados: fixed typo in help information

### PR DESCRIPTION
**Pull request by liupan1111**
Signed-off-by: Pan Liu <wanjun.lp@alibaba-inc.com>

### CODE REVIEW

**Comment by liupan1111** [comment]
@tchaikov Ping.

File: src/tools/rados/rados.cc [hunk 1]
```diff
@@ -83,7 +83,7 @@ void usage(ostream& out)
 "OBJECT COMMANDS\n"
 "   get <obj-name> [outfile]         fetch object\n"
 "   put <obj-name> [infile] [--offset offset]\n"
-"                                    write object write object start offset(default:0)\n"
```
Comment (tchaikov) [inline src/tools/rados/rados.cc]: better off adding a space before the opening parenthesis.

Comment (liupan1111) [inline src/tools/rados/rados.cc]: updated.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-06-15_17:39:27-rados-wip-kefu-testing---basic-smithi/ all issues are known ones.

**Comment by tchaikov** [comment]
retest this please

**Comment by tchaikov** [comment]
retest this please

**Comment by liupan1111** [comment]
retest this please

**Comment by liupan1111** [comment]
@tchaikov Jenkins passed.

**Pull request merged**